### PR TITLE
bid tracker updates

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Bid Tracker Sort By updates:
    * Bid Status -> Active Bids
    * Inactive Bids  (reverse of Active Bids)
    
“Bids Submitted and active” -> “Submitted Bids”

Lighter shade of gray when reviewing bids on my bid tracker that are no longer applicable(To See Dates)`bs_cd: C` to see

### CODE REVIEW

File: src/Components/BidTracker/BidTracker.jsx [hunk 1]
```diff
@@ -19,10 +19,12 @@ import BackButton from '../BackButton';
 const STATUS = 'status';
 const UPDATED = 'update_date';
 const LOCATION = 'position.post.location.city';
+const REVERSE_STATUS = '-status';
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/BidTracker.jsx:22]: 🤷‍♀️  I dont think we ever send back

File: src/sass/_bidTracker.scss [hunk 2]
```diff
@@ -172,8 +172,13 @@ $register-icon-offset: 44%;
...
 
 .bid-tracker-overlay-alert--closed {
-  background: $color-gray;
+  background: rgba(91, 97, 107, 0.8);
+  //option2: just keep the before color and change opacity on hover?
+  //   background: $color-gray;
 }
+//.bid-tracker-overlay-alert--closed:hover {
+//  opacity: 50%;
+//}
```
Comment (elizabeth-jimenez) [inline src/sass/_bidTracker.scss:181]: i prefer option 2 🙋‍♀️

**Review by scott062 — APPROVED** [review]
📕 🔐 🌂 📪  (they're all --closed icons)

**Comment by elizabeth-jimenez** [comment]
> 📕 🔐 🌂 📪 (they're all --closed icons)

oh.my.god. 😑 

😂

**Pull request merged**